src/the-nest/annotations.py

```diff
@@ -0,0 +1,36 @@
+x: int
+y: list[int | float]
+
+class Base:
+    a: int
+    b: str
+
+
+class Derived(Base):
+    pass
+
+
+def pretty_bin(n: int, *, sep: str = " ") -> str:
+    binary = f"{n:b}"
+    binary = binary.zfill((len(binary) // 4 + 1) * 4)
+    return sep.join(binary[i:i+4] for i in range(0, len(binary), 4))
```

Comment:
Add `from textwrap import wrap`.
```suggestion
    return sep.join(wrap(binary, 4))
```